common/src/edgeai_demo.cpp

```diff
@@ -0,0 +1,1147 @@
+                                            m_tidlInfObjs[i]->tidlObj.output, TIVX_TARGET_MPU_0);
+        }
+    }
+
+    /* Create mosaic module. */
+    for(uint i = 0; i < m_imgMosaicObjs.size(); i++)
+    {
+        /* Connect appropriate post process node to its appropriate mosaic node. */
+        for(uint j = 0; j < postProcMosaicConn[i].size(); j++)
+        {
+            int index = postProcMosaicConn[i][j];
+            m_imgMosaicObjs[i]->mosaic_input_arr[j] = m_postProcObjs[index]->dlPostProcObj.output_image.arr[0];
+        }
+        status = tiovx_img_mosaic_module_create(m_ovxGraph->graph,
+                                                &m_imgMosaicObjs[i]->imgMosaicObj,
+                                                m_imgMosaicObjs[i]->imgMosaicObj.background_image[0],
+                                                m_imgMosaicObjs[i]->mosaic_input_arr,
+                                                TIVX_TARGET_VPAC_MSC1);
+    }
+
+    /* Create display module if needed. */
+    if(m_displayObj != NULL)
+    {
+        status = tiovx_display_module_create(m_ovxGraph->graph,
+                                             m_displayObj,
+                                             m_imgMosaicObjs[m_dispMosaicIdx]->imgMosaicObj.output_image[0],
+                                             TIVX_TARGET_DISPLAY1);
+    }
+
+    /* Add graph parameter by node index */
+    graph_parameter_index = 0;
+    if(m_cameraObj != NULL)
+    {
+        status = add_graph_parameter_by_node_index(m_ovxGraph->graph, m_cameraObj->captureObj.node, 1);
+        m_cameraObj->captureObj.graph_parameter_index = graph_parameter_index;
+        graph_parameters_queue_params_list[graph_parameter_index].graph_parameter_index = graph_parameter_index;
+        graph_parameters_queue_params_list[graph_parameter_index].refs_list_size = m_cameraObj->captureObj.out_bufq_depth;
+        graph_parameters_queue_params_list[graph_parameter_index].refs_list = (vx_reference*)&m_cameraObj->captureObj.image_arr[0];
+        graph_parameter_index++;
+    }
+    for(uint i=0; i < m_multiScalerObjs.size(); i++)
+    {
+        if(status == VX_SUCCESS && m_multiScalerObjs[i]->isFirstNode)
+        {
+            status = add_graph_parameter_by_node_index(m_ovxGraph->graph, m_multiScalerObjs[i]->multiScalerObj1.node, 0);
+            m_multiScalerObjs[i]->multiScalerObj1.input.graph_parameter_index = graph_parameter_index;
+            graph_parameters_queue_params_list[graph_parameter_index].graph_parameter_index = graph_parameter_index;
+            graph_parameters_queue_params_list[graph_parameter_index].refs_list_size = 1;
+            graph_parameters_queue_params_list[graph_parameter_index].refs_list = (vx_reference*)&m_multiScalerObjs[i]->multiScalerObj1.input.image_handle[0];
+            graph_parameter_index++;
+        }
+    }
+
+    for(uint i = 0; i < m_imgMosaicObjs.size(); i++)
```

Comment:
Use stdint types (like uint32_t, etc).

Reply:
Updated the PR. Addressed all similar issues in the updated PR.